package.json

```diff
@@ -82,10 +82,10 @@
     "tslib": "^2.6.3",
     "whatwg-fetch": "^3.6.20"
   },
-  "packageManager": "pnpm@9.7.0",
   "msw": {
     "workerDirectory": [
       "web/.storybook/public"
     ]
-  }
+  },
+  "packageManager": "pnpm@9.8.0"
```

Comment:
I was thinking of setting [`update-notifier`](https://pnpm.io/npmrc#update-notifier) to false, as the pnpm team releases new versions pretty often and we don't need to stay at the freshest version at all times.

Too bad that dependabot cannot update pnpm for us, otherwise with `update-notifier` turned off we'd have to remember about the updates ourselves.

Reply:
I think disabling the notifications makes sense, 9.8.0 is the third release in August only. It's a bit annoying to see this notification that often.
Hopefully one day Dependeabot will [support updating `packageManager`](https://github.com/dependabot/dependabot-core/issues/4830).